doc/mundo.txt

```diff
@@ -223,7 +225,17 @@ be useful on large files and undo trees to speed up Mundo.
 Default: 1 (automatically preview diffs)
 
 ------------------------------------------------------------------------------
-3.11 g:mundo_verbose_graph                               *mundo_verbose_graph*
+
+3.11 g:mundo_auto_preview_delay                     *mundo_auto_preview_delay*
+
+This is the delay in milliseconds before a preview diff is automatically 
+rendered. The delay is reset whenever the cursor moves. Use this to speed up 
+undo tree traversal when automatic previews are enabled.
```

Comment:
Rather than this being a setting AND a delay value it should just be a delay value (say 100 or 200?)...I can't think of a reason we'd want to let the user turn this feature off...if they don't like it they can set a very small value, effectively turning it off.

Reply:
Agreed. 

After playing around with this option, ~250ms seems like a good choice, the preview delay is short but not so short that a user might scroll a little too slowly and trigger a preview, which may cause an annoying delay with larger files.
@simnalamburt any thoughts on a default?

Reply:
Sounds great - I tried 100 and 200 and was happy with both.